releasetool/commands/tag/python.py

```diff
@@ -56,44 +56,33 @@ def determine_release_tag(ctx: TagContext) -> None:
             f"head ref {head_ref}."
         )
         ctx.release_tag = click.prompt(
-            "What should the release tag be (for example, storage-1.2.3)?"
+            "What should the release tag be (for example, v1.2.3)?"
         )
 
     click.secho(f"Release tag is {ctx.release_tag}.")
 
 
-def determine_package_name_and_version(ctx: TagContext) -> None:
-    click.secho("> Determining the package name and version.", fg="cyan")
-    if "google-cloud-python" in ctx.upstream_repo:
-        match = re.match(
-            r"(?P<name>.+?)-(?P<version>\d+?\.\d+?(\.\d+)+)", ctx.release_tag
-        )
-        ctx.package_name = match.group("name")
-        ctx.release_version = match.group("version")
-        click.secho(
-            f"Package name: {ctx.package_name}, "
-            f"package version: {ctx.release_version}."
-        )
-    else:
-        match = re.match(r"(.+)?(?P<version>\d+?\.\d+?(\.\d+)+)", ctx.release_tag)
-        ctx.release_version = match.group("version")
-        click.secho(f"Package version: {ctx.release_version}.")
+def determine_package_version(ctx: TagContext) -> None:
+    click.secho("> Determining the package version.", fg="cyan")
+    # strip the leading 'v' from the tag
```

Comment:
Release-please will open PRs in the format `release-v1.2.3.` so:

`ctx.release_pr` is `release-v1.2.3`
`ctx.release_tag` is `v1.2.3`
`ctx.release_version` is `1.2.3`

Reply:
The best regex is no regex :)